features/step_definitions/metering.rb

```diff
@@ -1,23 +1,27 @@
 # helper step for metering scenarios
 require 'ssl'
+
 # For metering automation, we just install once and re-use the installation
 # unless the scenarios are for testing installation
 # checks that metering service is already installed
-Given /^metering service has been installed successfully(?: using (ansible|shell script|OLM))?$/ do |method|
+# shell => ./bin/hack.sh
```

Comment:
Is it still valid ?

Reply:
Then perhaps we need to update line 10, or else we are not able to use it.

Reply:
@liangxia we plan to phase that `hack.sh` script out.  That's why I took it out as an option.

Reply:
Or we should remove this line as well ?  It is confusing that we leave it there but removed it from line 10

Reply:
line removed... PTAL again. thx!